schedule/yam/test_cases/multipath.yaml

```diff
@@ -18,3 +15,5 @@ schedule:
     - console/force_scheduled_tasks
   system_validation:
     - console/validate_multipath
+test_data:
```

Comment:
You are adding test data here, but you have the same file defined as `YAML_TEST_DATA` in the related MR.

Reply:
Thanks, I removed the test data in yaml file.